### DISCUSSION TITLE
-webkit- prefixes

### DISCUSSION THREAD

**Discussion by Cyberchicken** [animate-css/animate.css#1469] [General]
I downloaded animate.css and I noticed a lot of -webkit- prefixes (835 lines).
I checked them on CanIUse and just one (out of 12) is used today:
-webkit-backface-visibility

Is there a reason to keep all the others?
-webkit-animation-delay
-webkit-animation-duration
-webkit-animation-fill-mode
-webkit-animation-iteration-count
-webkit-animation-name
-webkit-animation-timing-function
-webkit-keyframes
-webkit-transform
-webkit-transform-origin
-webkit-transition-duration
-webkit-transition-timing-function

Thank you

**Comment by eltonmesquita** [comment]
We probably just need to update browser-list and recompile the whole thing. Should be fixed in the next release
